test/test_tk205.py

```diff
@@ -89,6 +92,17 @@ def test_xlsx_template_creation():
     for rs in rss:
         tk205.template(rs,'build/templates')
 
+def test_json_yaml_translation():
```

Comment:
Maybe move this up to be with the other "translation" tests?

Reply:
Agreed and done.